docs/website/src/md-image/md-image.ts

```diff
@@ -0,0 +1,118 @@
+import { attr, boolAttr, ESLBaseElement, listen } from '@exadel/esl/modules/esl-base-element/core';
+import { DeviceDetector, getOffsetPoint, getTouchPoint, Point } from '@exadel/esl';
+import { memoize } from '@exadel/esl/modules/esl-utils/decorators/memoize';
+
+export class MdImage extends ESLBaseElement {
+  static is = 'md-image';
+  static zoomClass: string = 'md-image-zoom';
+
+  @attr({ dataAttr: true })
+  public src: string;
+
+  @boolAttr()
+  public inZoom: boolean;
+  @boolAttr()
+  public inDrag: boolean;
+
+  public scale: number = 2;
+  protected startPoint: Point;
+  protected startPosition: Point;
+
+  @memoize()
+  get $image() {
+    const originalImage = document.createElement('esl-image');
+    originalImage.setAttribute('data-src', this.src);
+    originalImage.setAttribute('lazy', '');
+    originalImage.setAttribute('mode', 'fit');
+    return originalImage;
+  }
+
+  @memoize()
+  get $closeButton() {
+    const button = document.createElement('button');
+    button.onclick = this.onClose.bind(this);
+    button.setAttribute('class', 'close-button');
+    button.innerText = 'X';
```

Comment:
Cosmetic: most likely `x` visual representative will depend on the browser's default font, so consider something more static